app/controllers/games_controller.rb

```diff
@@ -32,19 +34,16 @@ def edit
   # POST /games
   # POST /games.json
   def create
-    @game = Game.new(game_params)
-    @game.current_turn = 0
-    @game.black_player_id = current_player.id if @game.white_player_id == 0
-    
-    respond_to do |format|
+    ActiveRecord::Base.transaction do
```

Comment:
@montague Atomicity ftw, and I fully support where this was leading. Honestly this was the first time I've even seen a transaction attempt in one of the projects, so it was a refresher to see it in a PR.

If we were going to do it in this way, I would also recommend doing it all in model level vs in the controller. Transactions are out of scope in the context of a controller IMO. 

On second look, the timer creates kind of are too - because they are intrinsically linked to the game, and like you said - the game is invalid without them. They may be an opportunity for an `after_create`, which would also guarantee our atomicity because it would be in the context of the `save` transaction.